src/Compilers/VisualBasic/Portable/xlf/VBResources.cs.xlf

```diff
@@ -2891,8 +2896,8 @@
         <note />
       </trans-unit>
       <trans-unit id="ERR_LineContWithCommentOrNoPrecSpace">
-        <source>Line continuation character '_' must be preceded by at least one white space and must be the last character on the line.</source>
-        <target state="translated">Před znakem pokračování řádku _ musí předcházet aspoň jeden mezerový znak a musí se jednat o poslední znak na řádku.</target>
+        <source>Line continuation character '_' must be preceded by at least one white space and followed by a comment or must be the last character on the line.</source>
```

Comment:
this sentence is very ambiguous.  it could be legally read as:

(must be preceded by at least one white space and followed by a comment) or must be the last character on the line

or

must be preceded by at least one white space and (followed by a comment or must be the last character on the line).


Instead, consider having separate messages:

1. "Line continuation character '_' must be preceded by at least one white space."  For when there isn't a preceding whitespace.
2. "Line continuation character must be followed by a comment or must be the last character on the line.

Reply:
I agree but that requires changing the one error into 2 and the parent code has no idea which case it is. If false is returned some higher level function returns ERR_LineContWithCommentOrNoPrecSpace and that error is checked in lots of places.

Reply:
I also agree the sentence is confusing and could be interpreted either way. Improving the English is easier and less impact.